src/lzqgame.ts

```diff
@@ -90,14 +100,26 @@ async function hostCreateNewGame(
  * Two players have joined. Alert the host!
  * @param roomId The room ID
  */
-function hostPrepareGame(this: Socket, gid: string) {
+async function hostPrepareGame(
+    this: Socket,
+    roomId: string,
```

Comment:
roomId and gameId are both strings of the same value (stringified Mongo ObjectId). Should be only one param that represents both (roomId on client-side, gid on server-side).